src/components/popover/popover.ts

```diff
@@ -47,9 +47,12 @@ export class Popover extends Component<Props, SpreadsheetChildEnv> {
     // within our control and to avoid leaking into external DOM
     const horizontalPosition = `left:${this.horizontalPosition() - this.spreadsheetPosition.x}`;
     const verticalPosition = `top:${this.verticalPosition() - this.spreadsheetPosition.y}`;
-    const height = `max-height:${
+    const max_height = Math.max(
```

Comment:
```suggestion
    const maxHeight = Math.max(
```
We use camelCase for the variables